src/components/NavBar/index.jsx

```diff
@@ -1,8 +1,67 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { Link } from "react-router-dom";
 
 const NavBar = () => {
+  const [darkMode, setDarkMode] = useState(false);
+  const [openmenu, setOpenMenu] = useState(false);
+
+  useEffect(() => {
+    const hammenu=window.document.getElementById('hammenu');
+    if (openmenu)
+    {
+      hammenu.classList.remove('hidden');
+    }
+    else
+    {
+      hammenu.classList.add('hidden');
+    }
+  }, [openmenu]);
+
+  const toggleDarkMode = () => {
+    setDarkMode(!darkMode);
+  };
+
+  const open = () => {
+    setOpenMenu(!openmenu);
+  }
+
+
   return (
-    <div>NavBar</div>
+    <div className='sticky flex w-screen justify-center flex-col items-center'>
+      <div className='flex rounded-2xl shadow-[0px_6px_8px_rgba(0,0,0,0.25)] w-[90vw] mx-3 my-4 px-10 py-1 bg-[#F4F4F5] dark:bg-[#44403C] dark:text-white'>
+        <div className='flex w-1/4 monu font-normal px-2 py-2 text-xl items-center'>
+          <Link to='/'>UniShare</Link>
+        </div>
+        <div className='mobile:hidden flex w-3/4 flex-row-reverse items-center'>
+          <button className='h-7 bg-transparent mx-2 rounded-lg shadow-md' onClick={toggleDarkMode}>
```

Comment:
define height and width for the dark mode in terms of px: h-[32px] w-[32px] and use "grid place-items-center" to center the svg inside it

Reply:
Added height width and used flex box to center it.